iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/preprocessing/main/oozie_app/workflow.xml

```diff
@@ -119,6 +119,12 @@
 			<description>metadata extraction cache location, path pointing to
 				root cache directory holding meta.json file</description>
 		</property>
+		<property>
+            <name>referenceextraction_project_fundingclass_blacklist_regex</name>
+            <value>$^</value>
```

Comment:
Yeah I've read about multilines, we don't have them here. Anyway I've changed it to make it safe.

Also added:
```Set to 'a^' by default to guarantee nothing will be matched.```
comments.